test/system/252-quadlet.bats

```diff
@@ -1467,4 +1467,22 @@ EOF
 
     run_podman rmi $(pause_image)
 }
+@test "quadlet - entrypoint" {
```

Comment:
I think it would be cheaper in terms of CPU cycles when testing to add a test to `test/e2e/` instead.

Reply:
Yes, I agree. Since there is nothing to complex going on when setting this key, testing Quadlet alone in `test/e2e` should suffice. You can see other examples in https://github.com/containers/podman/blob/main/test/e2e/quadlet_test.go#L826